x-pack/plugins/triggers_actions_ui/public/application/sections/alert_details/components/alerts_route.tsx

```diff
@@ -36,11 +36,12 @@ export const AlertsRoute: React.FunctionComponent<WithAlertSummaryProps> = ({
   } = useKibana().services;
 
   const [alertSummary, setAlertSummary] = useState<AlertSummary | null>(null);
+  const [numberOfExecutions, setNumberOfExecutions] = useState(60);
 
   useEffect(() => {
-    getAlertSummary(rule.id, loadAlertSummary, setAlertSummary, toasts);
+    getAlertSummary(rule.id, loadAlertSummary, setAlertSummary, toasts, numberOfExecutions);
```

Comment:
nit -> since we are working in this area, I will prefer to differentiate between the initialization and the change of #executions. I think we can create an onChangeNumberOfExecutions with an event params so we can avoid to create a lambda function on the OnChange of the `EuiSuperSelect`. We can also try to remove the `eslint-disable`, in the past I used a useRef to know if the component was initialized or not and use a basic function to load the data, I think we can applied it here too.